cluster/saltbase/salt/docker/init.sls

```diff
@@ -104,9 +104,24 @@ lxc-docker-{{ docker_ver }}:
     - sources:
       - lxc-docker-{{ docker_ver }}: /var/cache/docker-install/{{ deb }}
 
+docker_load_kube_apiserver:
+  cmd.run:
+   - name: docker load -i /srv/salt/kube-bins/kube-apiserver.tar
+
+docker_load_kube_controller_manager:
+  cmd.run:
+   - name: docker load -i /srv/salt/kube-bins/kube-controller-manager.tar
+
+docker_load_kube_scheduler:
+  cmd.run:
+   - name: docker load -i /srv/salt/kube-bins/kube-scheduler.tar
+
 docker:
   service.running:
     - enable: True
+    - prereq_in: docker_load_kube_apiserver
```

Comment:
same applies for this section.  there is a different docker state above in the file when using RedHat os family.  I am testing with these prereq_in clauses in that section as well.